src/Entity/Invoice.php

```diff
@@ -172,6 +172,15 @@ class Invoice
      */
     private $localized = false;
 
+    /**
+     * @var \DateTime
+     *
+     * * @Exporter\Expose(label="label.date", type="datetime")
```

Comment:
I guess you want to add a new translation, correct? In that case we need to change the label later (and remove the second *)

Reply:
done